hw/xbox/smbus_xbox_smc.c

```diff
@@ -178,17 +179,50 @@ static uint8_t smc_receive_byte(SMBusDevice *dev)
     return 0;
 }
 
+static uint8_t smbus_avpack_to_reg(char *avpack)
+{
+    if (strcmp(avpack, "scart") == 0) {
+        return SMC_REG_AVPACK_SCART;
+    }
+    if (strcmp(avpack, "hdtv") == 0) {
+        return SMC_REG_AVPACK_HDTV;
+    }
+    if (strcmp(avpack, "vga") == 0) {
+        return SMC_REG_AVPACK_VGA;
+    }
+    if (strcmp(avpack, "rfu") == 0) {
+        return SMC_REG_AVPACK_RFU;
+    }
+    if (strcmp(avpack, "svideo") == 0) {
+        return SMC_REG_AVPACK_SVIDEO;
+    }
+    if (strcmp(avpack, "standard") == 0) {
+        return SMC_REG_AVPACK_COMPOSITE;
+    }
+    if (strcmp(avpack, "none") == 0) {
+        return SMC_REG_AVPACK_NONE;
+    }
+}
+
 static void smbus_smc_realize(DeviceState *dev, Error **errp)
 {
     SMBusSMCDevice *smc = XBOX_SMC(dev);
+    char *avpack;
 
     smc->version_string_index = 0;
+    /* pretend to have a composite av pack plugged in by default */
+    smc->avpack_reg = SMC_REG_AVPACK_COMPOSITE;
```

Comment:
nit: For code clarity and localness, why not move this into an `else` for the `if (avpack) {` check? Then it's clear that we'll pick either what was provided OR this one.

*(Has pros and cons, I'll leave final decision to others or you)*

Reply:
Let it be so. :slightly_smiling_face: